rest-api/src/main/resources/definitions/assessment.yml

```diff
@@ -100,6 +100,19 @@ properties:
     imageResource:
         $ref: ./image_resource.yml
         x-nullable: true
+    frameworkIdentifier:
+        type: string
+    jsonSchemaUrl:
+        type: string
+    category:
+        type: string
+    minAge:
+        type: integer
+    maxAge:
+        type: integer
+    additionalMetadata:
+        type: object
+        description: Additional metadata fields, expressed as a key-value blob.
```

Comment:
Any of these fields that can be null, should be marked with x-nullable: true.

Reply:
Would also be helpful to add descriptions, since this ends up in our API documentation.

Reply:
We actually talked about that amongst ourselves a couple weeks back. Alx wrote something in the Code Convention docs (https://sagebionetworks.jira.com/wiki/spaces/BRIDGE/pages/74219580/Code+Conventions) about how x-nullable seems to not do anything and recommended dropping it altogether. Jack and I decided to adopt that convention going forward.

Unless x-nullable does in fact show up somewhere that the client side uses.

Reply:
> Would also be helpful to add descriptions, since this ends up in our API documentation.

@nategbrown9 Could you please let me know whether these updated descriptions are correct and adequate? Especially frameworkIdentifier and jsonSchemaUrl because I'm not as familiar with what those represent.

Reply:
> > Would also be helpful to add descriptions, since this ends up in our API documentation.
> 
> @nategbrown9 Could you please let me know whether these updated descriptions are correct and adequate? Especially frameworkIdentifier and jsonSchemaUrl because I'm not as familiar with what those represent.

That looks good. Thanks.